### PR TITLE
Add user follow button support.

### PR DESCRIPTION
**Pull request by kxxoling**
Preview:
image

And my settings:

```
  "plugins": [
     "github-buttons"
   ],
   "pluginsConfig": {
     "github-buttons": {
       "buttons": [{
         "user": "kxxoling",
         "type": "follow",
         "size": "large",
         "width": "234"
       }]
     }
  }
```

### CODE REVIEW

**Comment by kxxoling** [comment]
Since this would break the former config, it's better to publish with a higher version, which is ``3.0.0``.

File: src/plugin.js [hunk 1]
```diff
@@ -16,21 +16,61 @@ require(['gitbook'], function (gitbook) {
...
+            size,
+            width,
+            height,
+            count
+        } = button;
+
+        var size = size || "large";
+        var width = width || (size === "large" ? "150" : "100");
+        var height = height || (size === "large" ? "30" : "20");
+        var count = typeof count === "undefined" ? "true" : "false";
```
Comment (azu) [inline src/plugin.js]: We should treat `count: true` setting as `var count = true`.

Can you change this to following?

```js
var count = typeof count === "boolean" ? String(count) : "false";
```

Comment (kxxoling) [reply src/plugin.js]: This is moved from your code: https://github.com/azu/gitbook-plugin-github-buttons/blob/master/src/plugin.js#L58 
and would be applied to both ``createButton`` and ``createUserButton``. Should I?

Comment (azu) [reply src/plugin.js]: > This is moved from your code:

Oh, sorry.

> would be applied to both createButton and createUserButton. Should I?

Yes, Could you please do it?

Comment (kxxoling) [reply src/plugin.js]: BTW, I think ``var count = typeof count === "boolean" ? count : false;`` would be more directly.

One more thing, should we regard non boolean (including ``undefined``) values as ``false`` or ``true``?

Comment (azu) [reply src/plugin.js]: > One more thing, should we regard non boolean (including undefined) values as false or true?

https://ghbtns.com/github-btn.html?user=twbs&repo=bootstrap&type=watch
Default value seem to be `false`.

Comment (azu) [reply src/plugin.js]: > var count = typeof count === "boolean" ? count : false;

I agree. (also need to update README.md)

```diff
      "buttons": [{
        "user": "azu",
        "repo": "JavaScript-Plugin-Architecture",
        "type": "star",
        "size": "large"
      }, {
        "user": "azu",
        "type": "follow",
        "width": "230",
-        "count": "false"
+        "count": false
      }]
```

**Comment by kxxoling** [comment]
Fixed and updated.

**Comment by azu** [comment]
@kxxoling Thanks!

**Pull request merged**